### PR TITLE
Initial CLI tests (usage)

### PR DESCRIPTION
**Pull request by arielshaqed**
Need (at least) `remove` command to be able to test `deploy`, `logs`.

### CODE REVIEW

File: cli/src/test/cli.spec.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import test from 'ava';
+import { Shell, success } from 'specshell';
+import * as R from 'ramda';
+
+const process = R.evolve({ out: (x: Buffer) => x.toString(), err: (x: Buffer) => x.toString() });
+
+test('cli with no args shows VERSION, USAGE, COMMANDS', async (t) => {
```
Comment (bergundy) [inline cli/src/test/cli.spec.ts]: That's not what this tests

File: cli/src/test/cli.spec.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  const { out, err, ...status } = process(await shell.run('./bin/run'));
+
+  t.true(success(status));
+  t.assert(err === '');
+});
+
+test('cli with no args and cli --help give same output', async (t) => {
+  const shell = new Shell();
+  const noArgs = process(await shell.run('./bin/run'));
+  const helpArg = process(await shell.run('./bin/run'));
```
Comment (bergundy) [inline cli/src/test/cli.spec.ts]: Where is the `--help` arg?

File: cli/src/test/cli.spec.ts (hunk 1 above)
Comment (arielshaqed) [reply cli/src/test/cli.spec.ts]: Sorry, pushed the pre-rebase version.

File: cli/src/test/cli.spec.ts (hunk 2 above)
Comment (arielshaqed) [reply cli/src/test/cli.spec.ts]: Added *both* of them.

File: cli/src/test/cli.spec.ts (hunk 1 above)
Comment (bergundy) [reply cli/src/test/cli.spec.ts]: Still wrong..
```suggestion
test('cli with no args exits successfully', async (t) => {
```

Comment (arielshaqed) [reply cli/src/test/cli.spec.ts]: Yeah, that test dropped an early assertion.  Restored that one.

**Comment by arielshaqed** [comment]
Thanks!

**Pull request merged**